core/net.go

```diff
@@ -575,6 +636,7 @@ func (n *OpenBazaarNode) SendChat(peerID string, chatMessage *pb.Chat) error {
 		return err
 	}
 	ctx, cancel := context.WithTimeout(context.Background(), n.OfflineMessageFailoverTimeout)
+	n.SendRelayedMessage(p, nil, &m) // send relayed message immediately
```

Comment:
`SendRelayedMessage` is called in `SendMessage` which is called below. Why call it twice?

Reply:
I'm not seeing where else this is called.